github/readmes

```diff
@@ -0,0 +1,9 @@
+# Project Readme and Description
```

Comment:
Please add `.md` suffix to the file and link it in the ToC in the main README.md of this project.

Reply:
Thanks, done!